components/net/resource_thread.rs

```diff
@@ -436,15 +464,16 @@ impl ResourceManager {
             "chrome" => from_factory(chrome_loader::factory),
             "file" => from_factory(file_loader::factory),
             "http" | "https" | "view-source" => {
+                let resource_grp = self.get_resource_group(load_data.pipeline_id);
```

Comment:
In the test file, the scheme for the iframe `src` URL is **file:///** whereas the loader which has the code changes for private/public resource group requires the scheme **http/https** would be leading to the failed test case. Does this seem logical???